src/services/users.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+class Users {
+  constructor() {
+    this.data = [];
+  }
+
+  getUsers() {
+    return this.data;
+  }
+
+  createUser(name) {
+    const maxId = Math.max(...this.data.map(user => user.id));
+    const newId = (this.data.length > 0) ? maxId + 1 : 1;
```

Comment:
Looks similar like in expenses service, not critical, but I prefer to move that function to utils, or something like that, to avoid duplicates